app/models/amr_data_feed_config.rb

```diff
@@ -2,15 +2,14 @@
 #
 # Table name: amr_data_feed_configs
 #
-#  access_type             :text             not null
 #  column_separator        :text             default(","), not null
 #  created_at              :datetime         not null
 #  date_format             :text             not null
 #  description             :text             not null
 #  handle_off_by_one       :boolean          default(FALSE)
 #  header_example          :text
 #  id                      :bigint(8)        not null, primary key
-#  local_bucket_path       :text             not null
+#  identifier              :text             not null
```

Comment:
This now drives the local and archive paths using an instance method for each rather than persisting them...